### DISCUSSION TITLE
Slot for alternative content in QIntersection

### DISCUSSION THREAD

**Discussion by Jille** [quasarframework/quasar#15752] [Ideas / Proposals] [answered]
Hi. I have a huge and slow page with lots of QSelects. To improve performance I've wrapped them in <QIntersections>. That helps, so thanks for QIntersection :)

However, it breaks using ctrl-f to search on the page.

I propose to add a second slot "alternative" on QIntersection that gets rendered if the content is outside the viewport. I'd use it to put the modelValue of my QSelect there so it can be found.

**Comment by rstoenescu** [comment, marked as answer]
Implemented a new slot ("hidden"). Will be available in Quasar v2.11.11 and v1.22.10

**Reply by Jille** [reply]
Awesome, thank you :D
